fast64_internal/oot/data/oot_data.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from .oot_object_data import OoT_ObjectData
+from .oot_actor_data import OoT_ActorData
+from .oot_getters import OoT_Getters
+
+
+class OoT_Common:
+    """Common Data used by other functions or classes"""
+
+    getters = OoT_Getters()
```

Comment:
I feel like you're reinventing the import system here,
currently you're doing
```py
# oot_getters.py
class OoT_Getters:
    """Common 'get' functions"""

    def getRoot(self, xmlPath: str) -> Element:
        ...

# somewhere else
from .oot_data import OoT_Common
OoT_Common.getters.getRoot(...)
```
which is afaict the same as the more pythonic/common:
```py
# oot_getters.py
def getRoot(self, xmlPath: str) -> Element:
    ...

# somewhere else
from . import oot_getters
oot_getters.getRoot(...)
```